### PR TITLE
Issue 2 view entries

### PR DESCRIPTION
**Pull request by amorabryan**
Regular Browser Recording Issue #2
Private Browser Recording Issue #2

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -38,5 +38,112 @@ $codeJournal.addEventListener('submit', function (event) {
...
+  return $firstEntry;
+}
+
+const $journalList = document.querySelector('.journal-list');
+
+document.addEventListener('DOMContentLoaded', function () {
+  const lastView = data.view;
+  const dataEntries = data.entries;
+  if (lastView) {
+    viewSwap(lastView);
```
Comment (shawnkost) [inline js/main.js]: This `if` statement isn't needed. `data.view` will always have some sort of string, so we should just be calling `viewSwap(lastView)` directly

**Review by shawnkost — CHANGES_REQUESTED** [review]
Nicely done Bryan! The Save button needs to be moved to the right side, and then one more change requested below

**Comment by amorabryan** [comment]
Revised Button Styling Issue #2

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**